plugins/cairo-lang-macro-stable/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "cairo-lang-macro-stable"
+version = "0.0.1"
```

```suggestion
version = "1.0.0"
```

wouldn't it make more sense for this particular crate?